### ISSUE TITLE
Mysql actions doesn't fail in case of sql db errors/exceptions

### ISSUE THREAD

**Issue by humblearner** [StackStorm-Exchange/stackstorm-mysql#4]
**mysql.select action**:

```
$ st2 run mysql.select connection="community" query="DELETE FROM user_registration WHERE email=123@123.com;"
.
id: 592382922f28d409a61d1c55
status: succeeded
parameters:
  connection: community
  query: DELETE FROM user_registration WHERE email=123@123.com;
result:
  exit_code: 0
  result: false
  stderr: 'No handlers could be found for logger "st2.st2common.util.loader"

    '
  stdout: '(1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''@123.com'' at line 1")

    '
```

**mysql.insert action**:

```
$ st2 run mysql.insert connection="community"  table="user_registration" data="email=blah@blah.com"
.
id: 592383592f28d409a61d1c5e
status: succeeded
parameters:
  connection: community
  data:
    email: blah@blah.com
  table: user_registration
result:
  exit_code: 0
  result: false
  stderr: ''
  stdout: '(1062, "Duplicate entry ''blah@blah.com'' for key ''email''")

    '
```

**Comment by humblearner** [comment]
Reason:  https://github.com/StackStorm-Exchange/stackstorm-mysql/blob/master/actions/insert.py#L30 and https://github.com/StackStorm-Exchange/stackstorm-mysql/blob/master/actions/select.py#L38 we are returning only the result not the exception. There are two ways to solve this: 

**Solution 1**:
Return both result and error
`return (False, e)`
```
$ st2 run mysql.insert connection="community"  table="user_registration" data="email=blah@blah.com"
.
id: 592385c92f28d409a61d1c6d
status: failed
parameters:
  connection: community
  data:
    email: blah@blah.com
  table: user_registration
result:
  exit_code: 0
  result: (1062, "Duplicate entry 'blah@blah.com' for key 'email'")
  stderr: ''
  stdout: ''
```

**Solution 2**:
Raise exception
 `raise Exception(e)`
```
$ st2 run mysql.insert connection="community"  table="user_registration" data="email=blah@blah.com"
.
id: 592385ec2f28d409a61d1c70
status: failed
parameters:
  connection: community
  data:
    email: blah@blah.com
  table: user_registration
result:
  exit_code: 1
  result: None
  stderr: "Traceback (most recent call last):
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py", line 259, in <module>
    obj.run()
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/runners/python_action_wrapper.py", line 155, in run
    output = action.run(**self._parameters)
  File "/opt/stackstorm/packs/mysql/actions/insert.py", line 17, in run
    return self.insert(table, data)
  File "/opt/stackstorm/packs/mysql/actions/insert.py", line 31, in insert
    raise Exception(e)
Exception: (1062, "Duplicate entry 'blah@blah.com' for key 'email'")
"
  stdout: ''
```

**Comment by LindsayHill** [comment]
Fixed with #5
